packages/edit-site/src/components/sidebar-dataviews/add-new-view.js

```diff
@@ -78,8 +78,7 @@ function AddNewItemModalContent( { type, setIsAdding } ) {
 		>
 			<VStack spacing="5">
 				<TextControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
Enable "Custom dataviews" in Gutenberg experiments. Go to Pages in the site editor and add a custom view.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/46932cd6-d26c-446c-b677-eea85b159ddc" alt="Dataviews Custom view modal, before" width="371">|<img src="https://github.com/user-attachments/assets/b0109601-c8e5-4a70-bc8c-54debf4a1360" alt="Dataviews Custom view modal, after" width="371">|